templates/oemr_ui/page_heading/partials/page_heading.html.twig

```diff
@@ -0,0 +1,26 @@
+{#
+The HTML rendering a Page Heading from OemrUI->pageHeading()
+
+@package OpenEMR
+@subpackage OemrUi
+
+@author Robert Down <robertdown@live.com>
+@copyright Copyright (c) 2023 Robert Down
+@copyright Copyright (c) 2023 Providence Healthtech
+#}
+<nav class="navbar navbar-light navbar-expand-sm bg-light pt-0">
+    {% block pageHeading %}
+        <span class="navbar-brand mb-0 h1">{{ heading }}</span>
+    {% endblock %}
+
+    {% block actionButtons %}
+        <div class="collapse navbar-collapse">
+            <div class="navbar-nav ml-auto" id="pageHeadingNav">
+                {% for a in buttonList %}
+                    <a href="{{ a.getHref() }}" id="{{ a.getId() }}" title="{{ a.getTitle() }}" {{ a.getAttributes() }} class="nav-link {{ a.getAnchorClasses()|join(' ') }} "><i class="{{ a.getIconClass() }}">&nbsp;</i>{{ a.getDisplayText() }}</a>
```

Comment:
Would escape here instead of in the upstream code:
```
<a href="{{ a.getHref()|attr }}" id="{{ a.getId()|attr }}" title="{{ a.getTitle()|attr }}" {{ a.getAttributes() }} class="nav-link {{ a.getAnchorClasses()|join(' ')|attr }} "><i class="{{ a.getIconClass()|attr }}">&nbsp;</i>{{ a.getDisplayText()|text }}</a>
```